### DISCUSSION TITLE
[Suggestion] Inpainting Conditioning Mask Strength for Automatic1111 WebUi

### DISCUSSION THREAD

**Discussion by MasterDenis** [TheLastBen/fast-stable-diffusion#357] [Ideas]
Hi, I recently got a peek on the Inpainting Conditioning Mask Strength option and I was wondering if it is possible to bring it over to the colab for testing,
Thanks in Advance!

**Comment by TheLastBen** [comment]
Hi, the feature is already available in the settings tab in A1111, it is included in the webui

**Reply by MasterDenis** [reply]
Hey Thanks, I am aware just curious if we could have it on the main page like other presently existing ones have, because some people need to use it constantly just like the other scales (especially in img2img).

**Reply by TheLastBen** [reply]
there is an option in the webui to bring it to the main page, watch this vid : https://www.youtube.com/watch?v=VdDhGjkKD1A
